src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -171,6 +175,58 @@ private void PublishCode()
             _methodCodeNode.InitializeDebugVarInfos(_debugVarInfos);
         }
 
+        private void EncodeEHInfo()
+        {
+            // TODO: Folding of the same blobs!!!!!!
```

Comment:
Do you mean the entire blob including UNWIND_INFO or this extra blob for EH only?
In either case, I think compiler should cache the data and detect its duplication. It's also doable if we want to emit such unique data in a custom section and emit reference/fix-up here (for this LSDA) instead. In fact, dwarf format lays EH data in different section than eh_frame.

Note COMDAT folding is for the same name (or contents) in different object files by the linker. So, this is not the case since we are emiting one object file.